Ncov sit rep 2020 03 21 (#222)
nextstrain/ncov: narratives/ncov_sit-rep_2020-03-20.md

---
title: Genomic analysis of COVID-19 spread. Situation report 2020-03-20.
authors:
  - Emma Hodcroft
  - Nicola Müller
  - Cassia Wagner
  - Misja Ilcisin
  - James Hadfield
  - Sidney M. Bell
  - Richard Neher
  - Trevor Bedford
authorLinks:
  - https://neherlab.org/emma-hodcroft.html
  - https://bedford.io/team/nicola-mueller/
  - https://bedford.io/team/cassia-wagner/
  - https://bedford.io/team/misja-ilcisin/
  - https://bedford.io/team/james-hadfield/
  - https://twitter.com/sidneymbell
  - https://neherlab.org/richard-neher.html
  - https://bedford.io/team/trevor-bedford/
affiliations: "Fred Hutch, Seattle, USA; Biozentrum, Basel, Switzerland; CZI, CA, USA"

> date: "2020 March 19"

Reviewer: Can you add in two lines between 21 & 22?
translators:
translatorLinks: 

This should not break the file, but will make my life easier when we do translations.

* [What you can do](https://nextstrain.org/community/narratives/nextstrain/ncov@ncov_sit-rep_2020-03-21/sit-rep_2020-03-20?n=11).
* [FAQ & common misconceptions](https://nextstrain.org/community/narratives/nextstrain/ncov@ncov_sit-rep_2020-03-21/sit-rep_2020-03-20?n=12).
* [Scientific credit](https://nextstrain.org/community/narratives/nextstrain/ncov@ncov_sit-rep_2020-03-21/sit-rep_2020-03-20?n=13).

<!-- This is right-side text -->
```auspiceMainDisplayMarkdown
# Executive summary

Here, we analyzed 713 publicly shared COVID-19 genomes. By comparing these viral genomes to each other, we can characterize how COVID-19 is evolving and moving around the world.

In this report, we highlight that the virus has been introduced and is circulating locally in many parts of the globe. This emphasizes the importance of rapid, widespread testing, as well as a strong focus on slowing local transmissions.

In this week's updates, we report:  

* Evidence for travel-related introductions to many parts of the world.  
* Many places with recent introductions can expect to see cases expand in 4-8 week timeframe. They should prepare now.   

> * Many places that have recently implemented social distancing measures can expect to see cases continue to accumulate in the short-term, but for new cases to plateau and decline in the medium- to long-term.

Reviewer: For clarity's sake - 
Replace: "...but for new cases to plateau and decline in the medium- to long-term."
With: "...but new cases will likely plateau and decline in the medium- to long-term."

  <a href="https://nextstrain.org/narratives/trees-background/"><img alt="cartoon of a phylogenetic tree" width="100" src="http://data.nextstrain.org/img_toy_alignment_mini.png"/> How to Read Phylogenies</a>
</div>

## Further Reading

* Summary of the SARS-CoV-2 outbreak on [Wikipedia](https://en.wikipedia.org/wiki/2019%E2%80%9320_Wuhan_coronavirus_outbreak).
* All case counts cited within this report are drawn from the [WHO](https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200318-sitrep-58-covid-19.pdf?sfvrsn=20876712_2) as of 2020-03-19.
```

<!-- ############ SLIDE BREAK ############# -->

<!-- This is left-side text 3 -->
# [A note on sampling](https://nextstrain.org/ncov/2020-03-19?c=country&r=country&d=map&p=grid&legend=closed)
We currently have sequences from samples taken in 36 countries across 6 continents. This is an incredible feat -- sequencing an unknown, large RNA virus in the midst of a pandemic is difficult, and is only possible through the incredible work and timely sharing of data by scientists and physicians around the world.
<br><br>

> While this data enables us to infer many useful characteristics of the outbreak and track its spread in real time, it's important to emphasize that our conclusions are limited by the available data. This happens because we expect that the true number of infections is far greater than the number of confirmed cases. Of these confirmed cases, only a fraction of viral samples are sequenced, and this fraction varies widely by country and region. Let's look at a few examples.  

Reviewer: Yes! Except:
"Of the overall number of cases, only a subset of patients are tested, and of those, only a subset of the viral genomes are sequenced."
&
"Which subset of the overall cases is tested and sequenced..." I think "number of" is unnecessary; this is clearer.

> While this data enables us to infer many useful characteristics of the outbreak and track its spread in real time, it's important to emphasize that our conclusions are limited by the available data. This happens because we expect that the true number of infections is far greater than the number of confirmed cases. Of these confirmed cases, only a fraction of viral samples are sequenced, and this fraction varies widely by country and region. Let's look at a few examples.  

Author: Zomg thank you. This short turnaround time can make it hard to step back from my own writing and realize when I've got bizarre wording in there. This is so much better :D

<!-- There is NO right-side text -->

<!-- ############ SLIDE BREAK ############# -->
<!-- This is left-side text 5 -->
# [Sequence data can cross-check travel histories](https://nextstrain.org/ncov/2020-03-19?legend=closed&&c=division_exposure&label=clade:A1a&d=tree,map)

Identifying where a given case was infected is important for understanding which areas are experiencing local transmission versus primarily travel-associated cases. Sequence data can help us differentiate between these scenarios.
<br><br>
Here, the tree is colored by the travel history of a case (when known). Mousing over the blue Canadian sequence in the middle, we see that they have a reported travel history to Europe. This infection groups with other European sequences, showing that this case almost certainly is a travel-related infection.

<!-- ############ SLIDE BREAK ############# -->
<!-- This is left-side text 6 -->
# [Sequence data can cross-check travel histories](https://nextstrain.org/ncov/2020-03-19?legend=closed&branchLabel=clade&c=division_exposure&d=tree,map&p=full)

> Zooming out, we can see that this isn't always so consistent.

Reviewer: Would consider adding `<br><br>` after this line for viz.

For example, mousing over the bright green Canadian sequence in the top right (near clade A7), we see that this case has a travel history to Iran.

> However, this sequence doesn't group anywhere near the rest of the cases with Iranian travel history! This makes it much less clear where this person was actually infected, despite the travel history.

Reviewer: Replace:
"This makes it much less clear..."
With:
"Thus, it is much less clear..."

<!-- There is no right side text -->

<!-- ############ SLIDE BREAK ############# -->

<!-- This is left-side text 7 -->
# [The virus has been introduced to nearly every country around the world](https://nextstrain.org/ncov/2020-03-19?legend=closed&branchLabel=clade&c=country&d=tree,map&p=grid)

On the tree, we see lots of this kind of intermingling of samples from different countries.

> This indicates that this kind of inevitable human movement has already introduced the virus to many parts of the globe.

Reviewer: "On the tree, we see lots of ~this kind of~ intermingling..."
"This indicates ~that this kind of~ inevitable human movement..."

<br><br>
In fact, the WHO reports confirmed cases in 159 out of 195 total countries in the world.
<br><br>

> However, it's important to keep in mind that not all introductions result in outbreaks.

Reviewer: "However, ~it's important to keep in mind that~ not all introductions..."

<!-- There is no right side text -->

<!-- ############ SLIDE BREAK ############# -->
<!-- This is left-side text 8 -->
# [Epidemics become evident ~4-8 weeks after introduction](https://nextstrain.org/ncov/2020-03-19?legend=closed&branchLabel=clade&c=num_date&d=tree&f_division=Washington&label=clade:B1&p=full)

> It may seem as though COVID-19 has suddenly exploded -- for many people, this outbreak may have gone from an abstract concern in other parts of the world to something that severely impacts our daily lives, seemingly overnight.

Reviewer: Consider making "for many people..." a brand new sentence. Also, "our" should be "their".

<br><br>
But, keep in mind that this has been brewing for awhile. It's not that the virus is being introduced to new places more often.

> Rather, it's that introductions that happened weeks ago started localized outbreaks that we're now becoming aware of.

Reviewer: For clarity,
Replace:
"It's not that the virus is being introduced to new places more often. Rather, it's that introductions..."
With: "The virus is not necessarily being introduced to new places more often. Rather, introductions..."

```

<!-- ############ SLIDE BREAK ############# -->
# [Once introduced, epidemics become evident ~4-8 weeks later](https://nextstrain.org/ncov/2020-03-19?legend=closed&d=tree,map&f_division=Washington&label=clade:B1&p=grid&r=location)

We can see a clear example of this in Washington state.
Mousing over the internal nodes at the root of the clade tells us that the virus was likely introduced once to this area in early- to mid-January 2020 ([methods](https://nextstrain.org/narratives/trees-background?n=6)).
<br><br>
Now, in mid-March (~8 weeks later), we see that a local outbreak is in full swing in this area. This is consistent with a doubling rate of about 4 days, assuming an exponentially growing population.  
<br>
<img src="https://github.com/nextstrain/ncov/raw/master/figures/wa_doubling-rate_2020-03-19.png" width="70%">

<!-- There is no right side text -->

<!-- ############ SLIDE BREAK ############# -->
<!-- This is left-side text 9-->
# [How outbreaks grow: an example](https://nextstrain.org/ncov/2020-03-19?legend=closed&d=tree,map&f_division=Washington,Utah&label=clade:B1&p=grid)

> We can also see early evidence that 'sparks' from this cluster have led to introductions in Utah. Following this pattern, we expect to see a local outbreak in Utah pick up within the next 4 weeks.

Reviewer 2: We don't have evidence for local spread in Utah, so I would leave this out. We also don't know where those two cases were infected.

<br><br>

> This is just one example. Most importantly, areas that aren't yet aware of local outbreaks should prepare now and start surveillance testing.  

Reviewer: Between "...example. Most..." add:
"There may have been other introductions into Utah (or other areas) that we cannot see on the tree because they were not sequenced."

<!-- There is no right side text -->

<!-- ############ SLIDE BREAK ############# -->
<!-- This is left-side text 10 -->
# [Mitigation measures take time, but they save lives](https://nextstrain.org/ncov/2020-03-19)

Following this same logic, there are likely many local transmission chains that we don't yet know about.
<br><br>
This means that even for areas that just implemented social distancing measures, we should still expect case counts to grow over the next few weeks.

> These increases do NOT indicate that these measures aren't working. It just takes time for already-infected individuals (and possibly their household members) to manifest symptoms, be treated, and recover. We'll also see a bump in the number of reported cases as testing becomes more widely available.

Reviewer: Replace: "We'll also see a bump in the number of reported cases as testing becomes more widely available."
With: "We also expect to see a bump in the number of reported cases as testing becomes more widely available."

  <a href="https://nextstrain.org/help/coronavirus/SARS-CoV-2"><img alt="illustration of a coronavirus" width="100" src="http://data.nextstrain.org/img_nCoV-CDC.jpg"/> Recent COVID-19 Outbreak Background </a>

  <a href="https://nextstrain.org/narratives/trees-background/"><img alt="cartoon of a phylogenetic tree" width="100" src="http://data.nextstrain.org/img_toy_alignment_mini.png"/> How to Read Phylogenies</a>
</div>

## Further Reading

* Summary of the SARS-CoV-2 outbreak on [Wikipedia](https://en.wikipedia.org/wiki/2019%E2%80%9320_Wuhan_coronavirus_outbreak).
* All case counts cited within this report are drawn from the [WHO](https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200318-sitrep-58-covid-19.pdf?sfvrsn=20876712_2) as of 2020-03-20.
```

<!-- ############ SLIDE BREAK ############# -->

<!-- This is left-side text 3 -->
# [A note on sampling](https://nextstrain.org/ncov/2020-03-19?c=country&r=country&d=map&p=grid&legend=closed)

> We currently have sequences from samples taken in 36 countries across 6 continents. This is an incredible feat -- sequencing an unknown, large RNA virus in the midst of a pandemic is difficult, and is only possible through the incredible work and timely sharing of data by scientists and physicians around the world.

Reviewer 2: ```suggestion
We currently have sequences from samples taken in 36 countries across 6 continents. This is an incredible achievement -- sequencing an unknown, large RNA virus in the midst of a pandemic is difficult, and is only possible through the incredible work and timely sharing of data by scientists and physicians around the world.
```

Below is a thought experiment showing how this played out in Wuhan. On the y axis we have space; on the x axis, we have time. Shaded area represents the number of cases.

"COVID-19 in Wuhan went from an index case in ~Nov 2019 to several thousand cases by mid-Jan 2020, thus going from initial seeding event to widespread local transmission in the span of ~10 weeks. We believe that international seeding events started to occur in mid-Jan. Thus we have a critical ~10 weeks from then to late-March to contain these nascent outbreaks before they become sizable." [- Trevor Bedford](https://twitter.com/trvrb/status/1226241284207038464), Feb 2020

These new outbreaks may, in turn, seed further introductions to other locations.

<img src="https://github.com/nextstrain/ncov/raw/master/figures/local-spark-expansion.jpeg" width="70%">

```

<!-- ############ SLIDE BREAK ############# -->
# [How outbreaks grow: an example](https://nextstrain.org/ncov/2020-03-19?legend=closed&d=tree,map&f_division=Washington&label=clade:B1&p=grid&r=location)

We can see a clear example of this in the data from Washington state.

> Mousing over root of the main clade shows that the virus was likely introduced to this area around the end of January 2020 ([methods](https://nextstrain.org/narratives/trees-background?n=6)); this is consistent with the first reported case on January 19, 2020.

Reviewer 2: Mousing over root of the main clade shows that the virus was likely introduced to this area between the end of January 2020 and mid February ([methods](https://nextstrain.org/narratives/trees-background?n=6)).

I would drop: "this is consistent with the first reported case on January 19, 2020.". There are a bunch of samples from China that are the same as WA1 and were sampled later than WA1. It could be two introductions into WA

> Mousing over root of the main clade shows that the virus was likely introduced to this area around the end of January 2020 ([methods](https://nextstrain.org/narratives/trees-background?n=6)); this is consistent with the first reported case on January 19, 2020.

Reviewer 2: ```suggestion
Mousing over root of the main clade shows that the virus was likely introduced to this area between the end of January and the middle of February 2020 ([methods](https://nextstrain.org/narratives/trees-background?n=6)).
```

<br><br>

> Now, in mid-March (~8 weeks later), we see that a local outbreak is in full swing in this area. This is consistent with a doubling rate of about 4 days, assuming an exponentially growing population.  

Reviewer 2: Now, in mid-March (~6 weeks later), we see

> Now, in mid-March (~8 weeks later), we see that a local outbreak is in full swing in this area. This is consistent with a doubling rate of about 4 days, assuming an exponentially growing population.  

Reviewer 2: ```suggestion
Now, in mid-March (~6 weeks later), we see that a local outbreak is in full swing in this area. This is consistent with a doubling rate of about 4 days, assuming an exponentially growing population.  
```
